e2e/testcases/otel_collector_test.go

```diff
@@ -54,12 +54,46 @@ const (
 	GCMMetricPrefix               = "custom.googleapis.com/opencensus/config_sync"
 )
 
+var DefaultGCMMetricTypes = []string{
```

Comment:
Good catch on the `resource_ns_count`, fixed.

The resource_fights_total and internal_errors_total are count metrics, which doesn't show up when no error condition presents. https://github.com/GoogleContainerTools/kpt-config-sync/pull/1237 intends to initialize these metrics when reconciler starts, which is not a perfect workaround since it generates a fake datapoint but get around the problem in http://b/318571531#comment32.

Reply:
Good to know. For posterity, we can add a comment to explain why they're not validated.

Reply:
Comment added.